resources/views/bank/nordigen/handler.blade.php

```diff
@@ -27,7 +30,7 @@
         // Logo URL that will be shown below the modal form.
         logoUrl: "{{ ($account ?? false) && !$account->isPaid() ? asset('images/invoiceninja-black-logo-2.png') : (isset($company) && !is_null($company) ? $company->present()->logo() : asset('images/invoiceninja-black-logo-2.png')) }}",
         // Will display country list with corresponding institutions. When `countryFilter` is set to `false`, only list of institutions will be shown.
-        countryFilter: false,
+        countryFilter: true,
```

Comment:
please explain this change. @turbo124 and I have descided when originaly implementing it to omit the country selection to reduce steps for the user to take.

Reply:
Both versions have their upsides. Without the country list it's one less click, but the list is highly polluted and scrolling is more difficult because one Bank may have dozens of Institutions in a single country.

With country filtering enabled, it's one more click but the bank lists are much easier to read and scroll through. It's also easy to see at a glance which institutions are supported in your country.

Santander is a good example where country filtering makes the most difference:
![image](https://github.com/user-attachments/assets/7a9e76f5-c7af-4d8f-a4af-38edec96a9a6)
There are 3 *Banco Santander*—in Spain, Brazil and Portugal—but there's no way to know without hovering the link and googling the institution code.

Likewise for *Santander*—one is Germany, the other UK—but without country filtering it's easy to get it wrong.

Reply:
good catch
@turbo124 I would therefore agree to introduce using the country select :)